api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 242**
+
+* [`POST /realm/profile_fields`](/api/create-custom-profile-field),
+[`GET /realm/profile_fields`](/api/get-custom-profile-fields): Added a
+new parameter `required`, which clients use to make field required to
+be filled by all the users in the organization.
```

Comment:
I'm pretty sure the `register` and `events` endpoints were updated as well, right?

Reply:
Yeah, I have added them now in "changelog.md".